javascript/browser-apis/reading-writing/blobs.md

```diff
@@ -33,34 +20,44 @@ links:
 ---
 ## Content
 
-A *blob* object represents an "almost file" object of raw data. It represents data that isn't in JavaScript's standard format. You use *FileReader* to read data from a blob.
+A binary large object, or *blob*, represents a file-like object of immutable raw data. It represents data that isn't in JavaScript's standard format. You use `FileReader` to read data from a blob.
```

Comment:
should we mention that binary objects are formed of binary numbers?

Reply:
Also "immutable" should be a footnote probably.